File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.go
func removeDuplicates(nums []int) int {
	i := 0
	for j := 1; j < len(nums); j++ {
		if nums[i] != nums[j] {
			nums[i + 1] = nums[j]
			i++
		}

	}

	return i + 1
}

File: Data Structures & Algorithms/maximum-subarray/submission-0.cpp
class Solution {
public:
    int maxSubArray(vector<int>& nums) {
        int result = nums[0], currentSum = result;
        for ( int i = 1; i < nums.size(); i++ ){
            currentSum = max(nums[i], currentSum + nums[i]);
            result = max(currentSum, result);
        }

        return result;
    }
};
